cmd/cli/cmd/manifest/generate/generate.go

```diff
@@ -17,10 +19,16 @@ import (
 func NewCmd() *cobra.Command {
 	var opts common.ManifestGenOptions
 	cmd := &cobra.Command{
-		Use:   "manifest-gen",
-		Short: "Manifests generation",
+		Use:   "generate",
+		Short: "OCF Manifests generation",
 		Long:  "Subcommand for various manifest generation operations",
 		Args:  cobra.MaximumNArgs(0),
+		Example: heredoc.WithCLIName(`
+			# To generate manifests interactively, run: 
+			<cli> manifest generate
+			# Then, select which manifests kinds you want to generate.
+			# If the Interface is selected, the Type kind toggles
+			# input and output Type generation for a given Interface.`, cli.Name),
```

Comment:
I've also added an Example description to highlight that there is an interactive command for generating multiple kinds of manifests.